README.md

```diff
@@ -38,7 +38,17 @@ To get started:
 1. At this point, you should be able to run the notebooks in this repo. Explore our notebooks on the following computer vision domains. Make sure to change the kernel to "Python (cv)".
     - [/classification](classification#notebooks)
     - [/similarity](similarity#notebooks)
-    
+
+
+The utilities in [`utils_cv`](utils_cv) is `pip`-installable from the
```

Comment:
i think that this could be clearer -- that the environment.yml will install the utils_cv package, but if you would like to install it in a separate environment outside of cvbp, you can just use the `pip install` method

Reply:
I am not convinced the way this is done here is the way to go. As I mentioned multiple times before, I think we should wait and then copy what the Recommenders folks do. I will leave comments on this PR today. For now, my current thinking is to (i) add the option to do a pip install of the utils_cv folder; (ii) but not change any of the existing files (such as the environment.yaml) and leave our current installation instructions as-is